components/style/properties/data.py

```diff
@@ -135,7 +144,7 @@ def __init__(self, style_struct, name, spec=None, animatable=None, derived_from=
 
 class Shorthand(object):
     def __init__(self, name, sub_properties, spec=None, experimental=False, internal=False,
-                 allowed_in_keyframe_block=True, alias=None):
+                 allowed_in_keyframe_block=True, alias=None, prefixes=None):
```

Comment:
I'd really like this to be called `extra_prefixes` instead of just `prefixes`, but don't worry if it's too much churn.